aspnetcore/signalr/authn-and-authz.md

```diff
@@ -153,7 +153,10 @@ var connection = new HubConnectionBuilder()
     .Build();
 ```
 
-Windows Authentication is only supported by the browser client when using Microsoft Internet Explorer or Microsoft Edge.
+Windows authentication is supported in Internet Explorer and Microsoft Edge, but not in all browsers. For example, in Chrome and Safari, attempting to use Windows authentication and WebSockets:
+
+* Fails.
```

Comment:
Maybe it would be better as a sentence than a list. "Windows auth with WebSockets fails and the client will then attempt to fall back to other transports which might work better with Windows auth." 🤷‍♀️

Reply:
@BrennanConroy  how about now?

Reply:
Yeah looks good! Thanks :)